sw/airborne/firmwares/rotorcraft/autopilot_static.c

```diff
@@ -173,8 +173,14 @@ void autopilot_static_periodic(void)
     SetCommands(commands_failsafe);
   } else {
     guidance_v_run(autopilot_in_flight);
-    guidance_h_run(autopilot_in_flight);
+    guidance_h_run(autopilot_in_flight);    
     SetRotorcraftCommands(stabilization_cmd, autopilot_in_flight, autopilot_motors_on);
+
+    if (lock_wings == 1 && autopilot_motors_on && commands[COMMAND_THRUST] > 0) {
```

Comment:
I totally agree this is not the nicest solution. The thing is that setting zero thrust before here also seems to influence the other actuators (like ailerons). I really just want to set main motor to standstill and leave other control as is. Also, I like it for every controller (so, make it independent whether we use INDI or PID or whatever). This seemed to be the only place.

Ideally I would rather do this directly from the module somehow, but I could not find a way to do so. (as the event functions are called at the wrong moment, and there is no obvious way to influence that). This combined with extra AP->FBW complexity made me decide to do it here...